### PR TITLE
Single-camera tracking feature

### PR DESCRIPTION
**Pull request by Jonas0126**
Given input frames from a camera and labeled car detections, output the corresponding IDs of cars in each frame. This is for single-camera multi-object car tracking.

### CODE REVIEW

File: README.md
Comment (b06b01073) [inline README.md:1]: You should append this file to the existing readme  file

**Review by b06b01073 — COMMENTED** [review]
Modify the readme file instead of create a new one

File: README.md
Comment (Jonas0126) [reply README.md:1]: How do I modify it? Should I directly add my content behind the original content?

Comment (b06b01073) [reply README.md:1]: 可以

**Pull request merged**